### DISCUSSION TITLE
能否将请求路由到集群中某个具体的实例中去?

### DISCUSSION THREAD

**Discussion by didoodidoo** [bfenetworks/bfe#1009] [Q&A (问题和回答)] [answered]
bfe能否支持通过配置或插件将请求路由到集群中的某个具体实例中去?

**Comment by mileszhang2016** [comment]
BFE的路由转发机制，
请参考 https://github.com/baidu/bfe-book/blob/version1/design/model/model.md

**Reply by mileszhang2016** [reply]
一般针对实例的需求是session sticky。

如果需求不是session sticky，那就应该把实例分到不同的集群了

**Reply by didoodidoo** [reply]
如果需要根据请求的哈希来将 请求路由到不同的实例中去的话,就只能把一个实例当做一个集群来处理了?

**Reply by mileszhang2016** [reply]
“需要根据请求的哈希来将 请求路由到（同一个实例池中的）不同的实例中”
这个需求是怎么出来的？能把具体的场景说一下吗？
这种使用方法和云原生的思想和不一致呀

**Reply by didoodidoo** [reply]
是一个查询的需求,需要最大化利用本地缓存,所以希望对请求做一致性哈希

**Comment by iyangsj** [comment, marked as answer]
@didoodidoo 见实例级别会话保持 

- 功能介绍：https://github.com/baidu/bfe-book/blob/version1/design/gslb/gslb.md#%E4%BC%9A%E8%AF%9D%E4%BF%9D%E6%8C%81

- 配置参考：https://www.bfe-networks.net/zh_cn/configuration/server_data_conf/cluster_conf.data/#gslb

（GslbBasic.HashConf.SessionSticky)

**Reply by didoodidoo** [reply]
收到,感谢
